catalog-info.yaml

```diff
@@ -478,6 +478,57 @@ spec:
 # SECTION END: Exhaustive tests pipeline
 # **************************************
 
+# ****************************************
```

Comment:
we don't need to keep catalog-info in the branch (main is the source of truth as my understanding) but if it doesn't confuse, it is okay.

Reply:
Yeah, i was thinking though that a clean commit without selecting certain files would be good. Going to let @v1v have a chance to chime in before merging. Thanks for the review!